sdk/eventhub/event-hubs/rollup.base.config.js

```diff
@@ -117,11 +117,14 @@ export function browserConfig(test = false) {
 
       nodeResolve({
         mainFields: ["module", "browser"],
-        preferBuiltins: false
+        preferBuiltins: false,
+        dedupe: ["buffer", "events", "util", "process", "assert"]
```

Comment:
As far as I know, the dependencies of Event Hubs and Service Bus libraries are very similar, therefore I would assume that the list of things to dedupe would be similar too.

Service Bus de-dupes only `buffer`, here we are de-duping much more. Do you know the parts of the code in Event Hubs that are different from Service Bus and therefore depends on these 4 extra packages?

Reply:
Right, not too sure why - but one differences is that the test code itself is referring to these modules in EventHubs, whereas in ServiceBus, the references are all mostly in the source. So likely, as we look into #3319 we may be able to get to how rollup ends with dupes and get it to avoid these potentially.

Reply:
- assert
     -  I see `assert` being used in the tests for Event Hubs which we don't in Service Bus, so that seems justified.
- process
     - `process` is used exactly the same way in Service Bus and Event Hubs, so not sure why we need to de-dupe that
- util and events
    - I don't see these used in the tests.

Can you remove `util`, `events` and `process` one by one, run the tests and confirm that you really need them de-duped?

Reply:
Yes, looks like these are still needed